nautobot_chatops/dispatchers/mattermost.py

```diff
@@ -403,11 +407,14 @@ def send_blocks(self, blocks, callback_id=None, modal=False, ephemeral=False, ti
             self.send_exception(mm_error)
 
     @BACKEND_ACTION_SNIPPET.time()
-    def send_snippet(self, text, title=None):
+    def send_snippet(self, text, title=None, ephemeral=None):
```

Comment:
If I'm reading the diff correctly, you're *only* adding `ephemeral` as an option to `MattermostDispatcher.send_snippet()` and `SlackDispatcher.send_snippet()`, but not to the other dispatchers? That seems incorrect to me - all dispatchers *must* have the same public API, even if they don't necessarily make use of certain optional parameters.